src/promptflow/promptflow/_sdk/data/executable/app.py.jinja2

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import sys
+
+from promptflow._cli._pf._connection import create_connection
```

Comment:
why do we need this could this be part of main?

Reply:
yeah, I also can move create_connection to "main.py"

Reply:
updated:
if move create_connectionto main.py, it will raise concurrent issues:
`sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 31324 and this is thread id 48884.`
![image](https://github.com/microsoft/promptflow/assets/26239730/e24fb51d-0e98-4236-bcae-b0249d37883b)

Reply:
I will leave create_connection in app.py for now and use streamlit to set connection key. like
![image](https://github.com/microsoft/promptflow/assets/26239730/a914433b-020f-469b-99bc-aef6115932e3)